docs/reference/cli-reference/cli-ref-push.md

```diff
@@ -109,3 +109,5 @@ nuget push foo.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -src https://customsou
 :: In the example below -SkipDuplicate will skip pushing the package if package "Foo" version "5.0.2" already exists on NuGet.org
 nuget push Foo.5.0.2.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -src https://api.nuget.org/v3/index.json -SkipDuplicate
 ```
+
+- For Azure DevOps Artifacts push examples, see [Azure Devops examples](https://docs.microsoft.com/en-us/azure/devops/artifacts/nuget/publish?view=azure-devops#examples).
```

Comment:
Isn't the name `Azure Artifacts`.

Reply:
Also when referring other MS docs, we use `/azure/devops` and we don't link the language specific resource.

Reply:
Sure, Addressed.